README.md

```diff
@@ -269,6 +269,21 @@ Once you have created the folder you can put all your custom icons there. But, a
 
 See [here the supported file extensions for an icon](https://github.com/vscode-icons/vscode-icons/blob/master/src/models/extensions/fileFormat.ts).
 
+#### Optional Angular icons
+Since **7.10.0**, along with the usual `Angular` icons, we shipped some optional icons for users wanting to have some specific icons for `component templates` and `component styles`.
+
+In order to enable them just add this to your settings:
+
+```js
+
+{ "icon": "ng_component_html", "extensions": ["component.html", ".component.htm"], "format": "svg", "disabled": true },
```

Comment:
There is no dot before the extensions property? And I think it's better to say it's disabled, otherwise we get tons of issues why it does not work because "it is not disabled by default"?